From 2b4dcfa8d6338a6fe2f3f59972a1052dc2d8f9c1 Mon Sep 17 00:00:00 2001
From: Alvaro-Karsz <alvaro.karsz@solid-run.com>
Date: Wed, 27 Oct 2021 13:44:23 +0300
Subject: [PATCH] Change page size from 64KB to 4KB.

Default page size from EVM defconfig is 64K, this commit changes it to 4K.
---
 arch/arm64/configs/am64xx-solidrun_defconfig | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/arch/arm64/configs/am64xx-solidrun_defconfig b/arch/arm64/configs/am64xx-solidrun_defconfig
index 5c24059ee..125e5c335 100644
--- a/arch/arm64/configs/am64xx-solidrun_defconfig
+++ b/arch/arm64/configs/am64xx-solidrun_defconfig
@@ -353,9 +353,9 @@ CONFIG_FUJITSU_ERRATUM_010001=y
 CONFIG_SOCIONEXT_SYNQUACER_PREITS=y
 # end of ARM errata workarounds via the alternatives framework
 
-# CONFIG_ARM64_4K_PAGES is not set
+CONFIG_ARM64_4K_PAGES=y
 # CONFIG_ARM64_16K_PAGES is not set
-CONFIG_ARM64_64K_PAGES=y
+# CONFIG_ARM64_64K_PAGES is not set
 # CONFIG_ARM64_VA_BITS_42 is not set
 CONFIG_ARM64_VA_BITS_48=y
 # CONFIG_ARM64_VA_BITS_52 is not set
-- 
2.25.1

